src/components/AdminForm/AdminForm.js

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from "react";
+import { NavLink, useLocation } from "react-router-dom";
+import "./AdminForm.css";
+
+export default function AdminForm() {
```

Comment:
Maybe split this up into LoginForm and Signup form, so the logic of the form has a dedicated place to live. 
If we try to make one component for both it adds needless complexity (if / else etc..)

Reply:
We have to do a big rewrite to work with the "real" backend anyway, so we can merge this and accept that dev can;t be merged with main until we are done with this process